src/app/file-browser/components/edit-tags/edit-tags.component.ts

```diff
@@ -269,4 +270,46 @@ export class EditTagsComponent
   close() {
     this.dialogRef.close();
   }
+
+  onArrowNav(event: KeyboardEvent, index: number) {
+    if (event.key === 'ArrowDown') {
```

Comment:
We should probably use `event.stopPropagation()` here, so that we don't even have to edit `file-viewer.component.ts` at all, as the event gets stopped before it propagates upward to parent components. We should do the same for the `setFocusToFirstTagOrButton` method as well.